exercises/rational-numbers/src/test/java/RationalTest.java

```diff
@@ -0,0 +1,260 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RationalTest {
+
+    // Helper methods
+
+    private static final double DOUBLE_EQUALITY_TOLERANCE = 1e-15;
+
+    private void assertDoublesEqual(double x, double y) {
+        assertEquals(x, y, DOUBLE_EQUALITY_TOLERANCE);
+    }
+
+    private void assertRationalsEqual(Rational r1, Rational r2) {
+        assertEquals(r1.getNumerator(), r2.getNumerator());
+        assertEquals(r1.getDenominator(), r2.getDenominator());
+    }
+
+    // Tests
+
+    @Test
```

Comment:
As per our [POLICIES](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests), only the first test should be used by default. All other tests should be `@Ignore`, in order to allow users to build up a solution.

This occurs a few times here, but I won't leave a comment on all of the exercises.